Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,124 @@
 import React from 'react';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
 import './App.css';
 
```

Comment:
I know this part was labor intensive but I'm glad you knocked it out and have your code very well organized here.